### ISSUE TITLE
Add ability to cache pages offline

### ISSUE THREAD

**Issue by Eeems** [dps/remarkable-wikipedia#14]
With the new online version of this, it would be great if users could cache a page for offline viewing

**Comment by cowpod** [comment]
true, although it's a bit redundant; we have zim files for offline use.
then theres also the issue of needing to clear the cache both manually and automatically for edits.

**Comment by dps** [comment]
I'm thinking of implementing this so it saves a PDF file to xochitl for later reading (and annotation!)

**Comment by dps** [comment]
Also note Issue #19 
https://github.com/dps/remarkable-wikipedia/issues/19

**Comment by cowpod** [comment]
Great idea! If I remember correctly KOreader has a similar feature. 
https://github.com/koreader/koreader
Off-topic: I'd like to say thank you for your amazing work; this works amazingly on the RM2!

**Comment by Eeems** [comment]
> true, although it's a bit redundant; we have zim files for offline use.

Right, but I'd prefer to not have to build it beforehand. I'd like to cache pages on the fly that I'd want to keep referencing.

**Comment by qubist** [comment]
This is also a good solution for us rM2 users who can't fit the whole English Wikipedia onto the internal memory and can't use an OTG SD card, but still want some offline-ready Wikipedia articles. I'm interested in this feature and would be happy to help with its development.
